### PR TITLE
Updating manifest incorporating changes that enable agg wts in heal discovery 

### PR DESCRIPTION
**Pull request by piotrsenkow**
…heal discovery page in preprod

Link to Jira ticket if there is one:
https://ctds-planx.atlassian.net/browse/HP-1383

### CODE REVIEW

File: preprod.healdata.org/portal/gitops.json [hunk 1]
```diff
@@ -118,6 +118,7 @@
     "analysis": false,
     "discovery": true,
     "discoveryUseAggMDS": true,
+    "workspaceTokenServiceRefreshTokenAtLogin": true,
```
Comment (mfshao) [inline preprod.healdata.org/portal/gitops.json:121]: should also enable `discoveryUseAggWTS` in here

File: preprod.healdata.org/metadata/aggregate_config.json [hunk 2]
```diff
@@ -194,7 +194,7 @@
         },
         "ICPSR": {
```
Comment (mfshao) [inline preprod.healdata.org/metadata/aggregate_config.json:195]: let's also remove the section about `SAMHDA` from this file https://github.com/uc-cdis/cdis-manifest/pull/7244/files#diff-67d6eecf0e0414ddd5ce26d56cf1ca73e4ad96d94b2b055ab83e8d99930dcaf4L94

File: preprod.healdata.org/portal/gitops.json (hunk 1 above)
Comment (piotrsenkow) [reply preprod.healdata.org/portal/gitops.json:122]: Thank you for pointing that feature flag @mfshao, I have addressed that in the latest commit

**Review by mfshao — APPROVED** [review]
There is a test in the CI that will fail, don't worry, I'll merge it when it is ready

**Pull request merged**